vignettes/estimate_static_severity.Rmd

```diff
@@ -52,7 +54,26 @@ that will likely be known to have had an outcome $t$ days after symptom onset
 maximum likelihood estimation to arrive at central estimates of the CFR or HFR,
 along with a confidence interval.
 
-Now we load the three packages we need using the following commands:
+This vignette outlines how to use _cfr_ to calculate disease severity in terms of CFR during an ongoing outbreak, using methods from @nishiura2009, using the the core functions within _cfr_.
```

Comment:
"using methods from @nishiura2009, using the the core" -> using methods from X in the core functions?

Reply:
Corrected to

> This vignette demonstrates how to calculate a static estimate of the severity of an outbreak using methods outlined @nishiura2009, and implemented in _cfr_.